content/blog/contributor-highlight-itay/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Contributor Spotlight: Itay Grudev"
+date: 2025-04-12T11:29:52
+draft: false
+image:
+    url: itay.jpg
+    attribution: 
+author: fdrees
+tags:
+ - helm
+ - grafana
+ - postgresql
+ - open-source
+summary: In a mini-series on this blog we highlight the work of the community. First up is Itay Grudev, component owner for the Operator chart and PostgreSQL Cluster Chart and the Grafana Dashboard for CloudNativePG.
+---
+
+Building and maintaining and open source project takes a village. In a mini-series on this blog we would like to highlight the work of our maintainers, component owners, and members of the larger community. 
+
+We start with [Itay Grudev](https://github.com/itay-grudev). Why? Because he was the first to reply: "You caught me at the perfect time, I was just drinking my morning coffee with not much else going on."
+
+Itay loves computers and has been fascinated with them since he was just a kid. It's no surprise then that he ended up working in Tech (and Kubernetes and DevOps in specific). He suspects that "some degree of autism helps getting along with computers too".
+
+## Why CloudNativePG
+Itay works as a Software Engineering Consultant at Verito Digital, a web agency. He first heard about CloudNativePG when he was looking for a well-designed PostgreSQL operator. "The company I worked for at the time became the second CloudNativePG adopter, after EDB came first."
```

Comment:
Even if it's quoting, I think the "after EDB came first" is a bit redundant.
```suggestion
Itay works as a Software Engineering Consultant at Verito Digital, a web agency. He first heard about CloudNativePG when he was looking for a well-designed PostgreSQL operator. "The company I worked for at the time became the second CloudNativePG adopter, after EDB."
```

Reply:
Was thinking again about the 80 column rule mentioned above.
If we did follow it, the suggestion here would have been much more surgical, instead of addressing the whole paragraph.